### PR TITLE
Match Attack Submenus

### PR DESCRIPTION
**Pull request by JoshDuMan**

### CODE REVIEW

File: src/190B20.c [hunk 1]
```diff
@@ -1133,7 +1133,69 @@ INCLUDE_ASM(s32, "190B20", func_802634B8);
...
+    s32 i;
+    s32 phi_s6;
+
+    for (i = 0; i < ARRAY_COUNT(battleStatus->submenuMoves); i++) {
+        battleStatus->submenuMoves[i] = 0;
+    }
+
+    battleStatus->submenuMoveCount = partner->staticActorData->level + 2;
+     // First move is changed each level
+    battleStatus->submenuMoves[0] = playerData->currentPartner * 6 +  0x7D + partner->staticActorData->level;
```
Comment (ethteck) [inline src/190B20.c:1151]:
```suggestion
    battleStatus->submenuMoves[0] = playerData->currentPartner * 6 + 0x7D + partner->staticActorData->level;
```

File: src/190B20.c [hunk 2]
```diff
@@ -1133,7 +1133,69 @@ INCLUDE_ASM(s32, "190B20", func_802634B8);
...
+
+    for (i = 1; i < battleStatus->submenuMoveCount; i++) {
+        battleStatus->submenuMoves[i] = playerData->currentPartner * 6 + 0x7F + i;
+    }
+
+    phi_s6 = 0;
```
Comment (ethteck) [inline src/190B20.c:1157]: FALSE

File: src/190B20.c [hunk 3]
```diff
@@ -1133,7 +1133,69 @@ INCLUDE_ASM(s32, "190B20", func_802634B8);
...
+                }
+            }
+
+            battleStatus->moveCategory = 5;
+            battleStatus->selectedItemID = partner->staticActorData->level;
+            battleStatus->currentTargetListFlags = move->flags;
+
+            player_create_target_list(partner);
+            if (partner->targetListLength != 0){
+                phi_s6 = 1;
```
Comment (ethteck) [inline src/190B20.c:1175]: TRUE

File: src/190B20.c [hunk 4]
```diff
@@ -1133,7 +1133,69 @@ INCLUDE_ASM(s32, "190B20", func_802634B8);
...
+                battleStatus->submenuEnabled[i] = 0;
+            }
+
+            if (gBattleStatus.flags2 & BS_FLAGS2_1000) {
+                battleStatus->submenuEnabled[i] = -1;
+            }
+
+    }
+
+    if (phi_s6 == 0) {
```
Comment (ethteck) [inline src/190B20.c:1193]: !phi_s6

**Review by ethteck — CHANGES_REQUESTED** [review]

**Review by Brotenko — COMMENTED** [review]
LGTM after implementing Ethan's requested changes!

**Comment by BowserSlug** [comment]
🚀 This PR matches 1 function (+0.01%) on `us`.

**Review by ethteck — APPROVED** [review]

**Review by Brotenko — APPROVED** [review]

**Review by nanaian — APPROVED** [review]

**Pull request merged**